scripts/config/propogateAuroChange.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Script used to propogate a specific change to all of our repositories by making identical PRs into every repository.
+
+# Required:
+#   - GitHub CLI >= 2.31.0
+#   - AlaskaAirlines GitHub PAT
+
+# Notes:
+#   - CANNOT be exectuted within another git repo
+
+# Parameters:
+#   `-b`: Name of branch PRs will be made from.
+#   `-t`: Title of PR that will be opened
+#   `-c`: Commit message for standard change
+
+# Parse Command Line Arguments
+while getopts b:t:c: flag
+do
+    case "${flag}" in
+        b) branchName=${OPTARG};;
+        t) prTitle=${OPTARG};;
+        c) commitMessage=${OPTARG};;
+    esac
+done
+
+# Validate all necc variables are defined
+if [ -z "$branchName" ]; then
+  echo "branchName undefined, please define using -b flag.";
+  exit 1;
+fi
+if [ -z "$prTitle" ]; then
+  echo "prTitle undefined, please define using -t flag.";
+  exit 1;
+fi
+if [ -z "$commitMessage" ]; then
+  echo "commitMessage undefined, please define using -c flag.";
+  exit 1;
+fi
+
+# Get All Auro Repositories
+repoOwner="AlaskaAirlines"
+repos=$(gh repo list $repoOwner \
+        --source \
+        --visibility=public \
+        --json name \
+        --jq .[].name \
+      | grep "auro-" \
+      );
+
+# Clear Previous Workspace
+rm -rf ./workspace
```

Comment:
This feels like brute force. What if the repo is already using a `./workspace` dir?